CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ AtomVM is open to any contribution.
 However before contributing, please read carefully our [Code of Conduct](CODE_OF_CONDUCT.md) and
 the following contribution guidelines.
 
-Please, also make sure to understand the [Apache 2.0 license](LICENSE.md) and the
+Please, also make sure to understand the [Apache 2.0 license](LICENSES/Apache-2.0.txt) and the
```

Comment:
`LICENSE.md` and `LICENSES/Apache-2.0.txt` are the same file. There is no point in changing this.

Reply:
I changed it because the link was broken, but I see now the file name is `LICENSE` (no file type on the end.) The `LICENSE` file is plain text, and not in markdown. I tried linking to the file without an extension, and in the html it turns into a download link, rather than just displaying the document.